tests/test_indefinite_freeze_attack_old.py

```diff
@@ -36,7 +36,6 @@
   metadata without the client being aware.
 """
 
-import datetime
```

Comment:
Why this change in an "_old.py" file?

Reply:
I just saw it was unused and decided to remove it, but I can return it if it looks strange.

Reply:
🤷  Probably not worth the effort. :) Let's leave it there.